packages/gui/src/components/nfts/NFTCard.tsx

```diff
@@ -30,18 +30,20 @@ export type NFTCardProps = {
   selected?: boolean;
   canExpandDetails: boolean;
   availableActions: NFTContextualActionTypes;
+  isOffer: boolean;
```

Comment:
The property name should reflect the behavior that it defines, not the origin of the property. Since the goal is to prevent displaying the thumbnail when viewing an offer, I would suggest a name like `disableThumbnail`

Reply:
Done.